llvm/lib/Transforms/Instrumentation/MemorySanitizer.cpp

```diff
@@ -3278,22 +3278,49 @@ struct MemorySanitizerVisitor : public InstVisitor<MemorySanitizerVisitor> {
     setOrigin(&I, getOrigin(Op));
   }
 
-  void handleCountZeroes(IntrinsicInst &I) {
```

Comment:
rename seems unrelated? i don't like the new name tbh

Reply:
Yes, it's a drive-by change. The old name is misleading, because it handles instructions that do not count all the zeros (unlike llvm.ctpop, which counts all the ones, regardless of whether they are leading, trailing, or middle). I'm open to other naming suggestions.